### ISSUE TITLE
Make latest version of Meme available on GitHub

### ISSUE THREAD

**Issue by rdebeasi** [voxmedia/meme#27]
The version of Meme used on SBNation includes a variety of cool new features that don't seem to be available on GitHub: 
- Basic Markdown support
- "Design" panel with a variety of new background and color options
- Vertical Alignment and Letter Spacing
- Uppercase heading/source options

Is there any chance of this new version of Meme making it to GitHub? Thanks very much in advance!

**Comment by mneznanski** [comment]
Agreed. That would be very cool. Looks like some things that [cardkit](https://github.com/times/cardkit) added, but not needing to completely change platforms would be nice.

**Comment by ryanmark** [comment]
We have no plans to do this. It may get taken up as a off hours or hackathon project, but no promises.

So here's the story: As we've built more tools for our editorial teams, we needed to wrap these things up together into a single thing to make it easier for our folks to find and easier for us to support and document. The downside is that we ended up forking this meme tool and slowly improving it with general improvements and Vox Media specific features.

Clearly this is a crappy way to do open source.

With our Autotune project, we're slowly moving towards a workflow that allows us to open-source more of our stuff without doubling our work load, but we're not there yet.

**Comment by mneznanski** [comment]
Fair enough. Thanks for the reply!
On Sep 11, 2015 5:36 AM, Ryan Mark notifications@github.com wrote:We have no plans to do this. It may get taken up as a off hours or hackathon project, but no promises.

So here's the story: As we've built more tools for our editorial teams, we needed to wrap these things up together into a single thing to make it easier for our folks to find and easier for us to support and document. The downside is that we ended up forking this meme tool and slowly improving it with general improvements and Vox Media specific features.

Clearly this is a crappy way to do open source.

With our Autotune project, we're slowly moving towards a workflow that allows us to open-source more of our stuff without doubling our work load, but we're not there yet.

**Comment by jpvalery** [comment]
That's sad to read :(

I couldn't find how to change the defaut size of the image generated (i changed in meme.js but with no effects).
Any tips ?

(and thanks @mneznanski for quoting cardkit. this is now my backup plan if I can get this one to work as I'd like :) )

**Comment by jpvalery** [comment]
Glad to see it updated :)
